onadata/apps/api/viewsets/open_data_viewset.py

```diff
@@ -97,7 +97,18 @@ def get_ordered_repeat_value(key, item, index):
                         item = get_ordered_repeat_value(key, item, index)
                         flat_dict.update(item)
                 else:
-                    flat_dict[key] = value
+                    try:
+                        qstn_type = xform.get_element(key).type
+                        if qstn_type == "select all that apply":
```

Comment:
Please use the [common tag](https://github.com/onaio/onadata/blob/3368ec5edfe6d0789c54043d4575dcbeaaf7521c/onadata/libs/utils/common_tags.py#L87)

Reply:
Done

Reply:
I can't seem to see this change @WinnyTroy